lib/tasks/cuttlefish.rake

```diff
@@ -16,6 +16,18 @@ namespace :cuttlefish do
     Email.all.each {|email| email.update_status! }
   end
 
+  desc "Archive all emails created more than 6 months ago"
+  task :archive_6_month_old_emails => :environment do
```

Comment:
I'd just call this something like `autoarchive`. That way if we decide to change it to say, 3 months, then we don't need to change job names everywhere.